docs/self-managed-operators.md

```diff
@@ -0,0 +1,16 @@
+# Self managed operators that requires CRDs and roles
+
+Compliant kubernetes only allow end users to install pre-approved operators that require CRDs.
+
+Pre-approved operators with CRDs:
+
+- Sealedsecrets
+- MongoDB
+
+## Limitations
+
+Compliant kubernetes do not allow end users to install clusterRoles and clusterRoleBindings.
```

Comment:
```suggestion
Compliant Kubernetes do not allow end users to install ClusterRoles and ClusterRoleBindings.
```

Sorry, my OCD. :smiley: 

Similar edits in different files would be appreciated but are of course not hugely needed.